content/blog/2023/2022blog.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "2022 Progress!"
+author: "ghostway"
+published: 2023-01-14
+---
+
+2022 has been a great year for Leela. A lot of new contributors appeared and made significant improvements; overall, Leela has become considerably stronger and more interesting.
+<!--more-->
+
+While 2021 had [ceres](https://lczero.org/blog/2021/01/announcing-ceres/) written and small (but exciting) improvements to the training pipeline,
```

Comment:
Style comment (feel free to ignore this and all further style comments):

* With "while" is sounds a bit like a opposition/contrast in a sentece, not a mere reminder.
* I'm not sure whether it makes sense to mention 2021 at all.
* But if yes, something like "In 2021, Ceres engine was announced, together with some small but important changes to our training pipeline"
* Also, if keeping the 2021 part of the sentence, I it may be worth mentioning ONNX support that appeared in 2021 and enabled quicker experimentation (but really started to be used at full potential in 2022).

Also, technically Ceres was written in 2020, not 2021.

Also (also of course totally optional), consider cleaning up the text through ChatGPT, it really helps (not necessary as a final result, but surely as an inspiration). Prompt would be something like "rephrase this paragraph for a blog post in a relaxed, friendly, and not too formal or too exciting style".

Reply:
I removed ceres, heh. I should probably remove the 2021 thing then